src/discussions/learners/LearnerPostsView.jsx

```diff
@@ -79,7 +79,9 @@ function LearnerPostsView({ intl }) {
       </div>
       <div className="bg-light-400 border border-light-300" />
       <div className="list-group list-group-flush">
-        {postInstances}
+        {postInstances(pinnedPosts)}
+        {pinnedPosts.length > 0 && <div className="pt-1 bg-light-500 border-top border-light-700" />}
```

Comment:
interstitial div cause accessibility issue when traversing a list using keyboard, I will test this branch locally before approval.

Reply:
Sure